docs/manual/02_Quick_start.md

```diff
@@ -0,0 +1,151 @@
+# Quick start
+
+This chapter will show you how to install and configure Clusterous so that you can start using it.
+
+## Installation
+
+### Preparation
+To install Clusterous, you need Python 2.7 and the associated package management tool "pip". Many systems already have Python and pip installed, so enter the following on the command line to check:
+
+    python --version
+
+If you do have the correct version of Python installed, the above command should simply output someething like `Python 2.7.9`. If Python is not installed, use your system's package management tool to install Python 2.7 and run the above command again.
+
+Next, ensure you have pip installed:
+
+    pip --version
+
+If you do have pip installed, you will pip's version number. If not, install pip using your system's package management tool. General instructions for obtaining pip are available here: (https://pip.pypa.io/en/stable/installing/).
+
+**Note on Python versions**: Some systems may have Python 3.x installed. Python 3.x is a separate, incompatible version of Python that cannot run Clusterous. You can however have both Python 2.7 and 3.x installed on the same system. If you have both versions, the `pip` command may by default point to the Python 3.x version, which can be confirmed by running `pip --version`. Once you have Python 2.7 installed, the appropriate version of pip can be run as `pip2`. You can verify this with:
+
+    pip2 --version
+
+Make sure you use `pip2` to install Clusterous.
+
+### Clusterous
+To install Clusterous, you need to obtain the .tar.gz installation package file. A copy of `clusterous-0.5.0.tar.gz`can be found on [Clusterous' GitHub releases page](https://github.com/sirca/clusterous/releases) under "Downloads". Alternatively, use the [direct download link for Clusterous 0.5.0](https://github.com/sirca/clusterous/releases/download/v0.5.0/clusterous-0.5.0.tar.gz).
+
+Once you have downloaded the installer, change to the directory where you downloaded it (or copy the installer to your current directory) and install Clusterous using the following command.
+
+    sudo pip install clusterous-v0.5.0.tar.gz
+
+Note that pip requires the file to be zipped. Do not extract the installer file into a directory.
+    
+
+### Verify
+
+To verify that Clusterous is installed, try:
+
+    clusterous --help
+    
+And you should see Clusterous' help output. Like a typical Unix command line tool, it should show you the list of commands and options.
+
+## Configuring
+Clusterous creates clusters on your own AWS account, so before you start using Clusterous, you need provide your AWS credentials and run the interactive Clusterous setup wizard.
+
+The AWS keys are in the form of an Access Key ID and Secret Access Key, and are provided to you by AWS when you create an IAM user in your AWS account. You may refer to this manual's guide [TODO: LINK] to preparing your AWS account for use. Your AWS credentials will typically look something like:
+
```

Comment:
The link is for the AWS section of the manual, which doesn't exist yet. I'll add a comment to that task to ensure this link is updated.